apisix/plugins/jwt-auth.lua

```diff
@@ -48,7 +48,11 @@ local schema = {
         hide_credentials = {
             type = "boolean",
             default = false
-        }
+        },
+        key_claim_name = {
+            type = "string",
```

Comment:
Add a limitation about the length, the minimum length should be 1

empty string isn’t acceptable